SubEthaEdit-Mac/Source/TCMMillionMonkeys/TCMMMLoggingState.h

```diff
@@ -22,7 +22,7 @@
     unsigned long insertedCharacters;
     unsigned long selectedCharacters;
     NSMutableArray *I_statisticsData;
-    TCMMMSession *I_MMSession;
+    __weak TCMMMSession *I_MMSession;
```

Comment:
Yes, all ivars need a special eye on them again after a big arc merge, it's easy to overlook those generated strong retain cycles that were solved differently before. Also the "new" debug memory graph tool (in the debugger next to the view debugger) shows those cycles quite nicely.

Reply:
Thanks for the hint regarding the memory tool (i used good ol' Instruments until then)